CPC-CHARTER.md

```diff
@@ -176,25 +176,17 @@ The CPC Voting Members will vote on candidates as described in the [Elections][]
 
 ## Section 9. Decision Making
 
-For internal Project decisions, Collaborators shall operate under Lazy Consensus.
-The CPC follows a [Consensus Seeking][] decision making model.
-When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+The CPC follows a Lazy Consensus Seeking decision-making model inspired by the [Apache Software Foundation's model][ASF community].
 
-If an agenda item cannot reach a consensus a CPC member can call for either a closing vote or a vote to table the issue to the next meeting.
-Votes shall follow the procedure described in the [Voting][] section of this charter.
+Lazy consensus is achieved by making a proposal in a forum that will reach all members of the group who should have the opportunity to participate in reaching consensus and waiting for an appropriate amount of time given the nature of the decision for anyone to object. It is not necessary to get explicit approval to proceed from all members, silence indicates consent. However, objections must be resolved in order to achieve consensus.
 
 ## Section 10. Voting
 
-The CPC follows a [Consensus Seeking][] decision making model.
-When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+Voting should only be done as a last resort.
 
-If an agenda item cannot reach a consensus a CPC member can call for either a closing vote or a vote to table the issue to the next meeting.
-The call for a vote must be seconded by a majority of the CPC or else the discussion will continue.
+If an agenda item cannot reach consensus after a reasonable period of discussion, a voting member can call for objections to be overruled through an asynchronous vote. If this call is seconded by another voting member, an asynchronous vote must be organized. For the objections to be overruled, at least 2/3 of all voting members must vote in favor of overruling the objections.
```

Comment:
Can you remove references to "agenda item" I had specifically removed those references as I don't think we want to imply that our concensus seeking is done in meetings or even requires one.  Anywhere is says agenda item I think it should say "proposal"

Reply:
Good catch!